samples/bluetooth/mesh/light/prj.conf

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# General configuration
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_SETTINGS=y
+CONFIG_HWINFO=y
+CONFIG_DK_LIBRARY=y
+CONFIG_LOG=y
+CONFIG_LOG_PRINTK=y
+
+# Bluetooth configuration
+CONFIG_BT=y
+CONFIG_BT_DEVICE_NAME="Mesh Light"
+CONFIG_BT_TINYCRYPT_ECC=y
+CONFIG_BT_L2CAP_TX_MTU=69
+CONFIG_BT_LL_SW_SPLIT=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_SETTINGS=y
+
+# Disable unused Bluetooth features
```

Comment:
Would it perhaps be possible to get some different BT_CTLR defaults when MESH is enabled? It would make it easier if you always wish to disable these features for MESH.